src/services/kratos/index.types.d.ts

```diff
@@ -25,3 +25,8 @@ type KratosPublicMetadata = {
 type KratosAdminMetadata = {
   phoneMetadata: PhoneMetadata | undefined
 }
+
+type FlowId = string & { readonly brand: unique symbol }
```

Comment:
I saw this type being used in domain code so it should not be defined here. Serivces depend domain not the other way around.
Apart from that I also don't like the naming (as commented elsewhere).